src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -71,6 +74,82 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
   }
   useNetworkStatus(startPolling, stopPolling);
 
+  const columnsTemplate: Array<ColumnProps<TaskResult>> = [
+    {
+      title: "Name",
+      dataIndex: "displayName",
+      key: TaskSortCategory.Name,
+      sorter: {
+        multiple: 4,
+      },
+      width: "40%",
+      className: "cy-task-table-col-NAME",
+      render: (name: string, { id }: TaskResult): JSX.Element => (
+        <TaskLink taskName={name} taskId={id} />
+      ),
+    },
+    {
+      title: "Patch Status",
+      dataIndex: "status",
+      key: TaskSortCategory.Status,
+      sorter: {
+        multiple: 4,
+      },
+      className: "cy-task-table-col-STATUS",
+      render: renderStatusBadge,
+    },
+    {
+      title: "Base Status",
+      dataIndex: "baseStatus",
+      key: TaskSortCategory.BaseStatus,
+      sorter: {
+        multiple: 4,
+      },
+      className: "cy-task-table-col-BASE_STATUS",
+      render: renderStatusBadge,
+    },
+    {
+      title: "Variant",
+      dataIndex: "buildVariant",
+      key: TaskSortCategory.Variant,
+      sorter: {
+        multiple: 4,
+      },
+      className: "cy-task-table-col-VARIANT",
+    },
+  ];
+  if (sorts.length === 0) {
+    // works around a quirk/bug in the antd table where default sorts don't
+    // get updated on subsequent renders
+    columnsTemplate[1].defaultSortOrder = "ascend";
+    columnsTemplate[2].defaultSortOrder = "ascend";
```

Comment:
1. This bit of logic is not necessary. It should be accomplished by you just updating the query params here. https://github.com/evergreen-ci/spruce/pull/591/files#diff-9dac9e5fbcce9bdaff0aa2c7996ad847f32fdcca0c789b9dd1286cf659cef383R57
And then letting the sortOrder be inferred from the query params instead of manually setting them